### DISCUSSION TITLE
Where do I create requests for improving existing mdn articles?

### DISCUSSION THREAD

**Discussion by DanKaplanSES** [mdn/mdn-community#513] [Content] [answered]
Here's the context: I'd like to create an issue re: the [nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) and [nth-of-type](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type) pages. Specifically, I'd like to propose they mention and include examples of how `nth-child` and `nth-of-type` differ. 

When I tried to create a new issue I was presented with this:

![image](https://github.com/mdn/content/assets/373675/4c3e4e38-ef18-4edd-8eae-877985bbc638)

I don't know which one I should use to propose my suggestion:

1.  ![image](https://github.com/mdn/content/assets/373675/89ed474c-7786-4219-ba8b-05cb745a3611)
  Should I use this one? It is a content issue.
1. ![image](https://github.com/mdn/content/assets/373675/89c208e1-38a2-4230-8c7e-0a2e06645b86)
  This one obviously doesn't fit.
1. ![image](https://github.com/mdn/content/assets/373675/e211f7f2-b0b8-46e4-9e7f-3fc1883f167b)
  I think this is saying that it should only be used by clicking a link from a rendered MDN page? If so, maybe it could be more explicit about that, mentioning it's for automated purposes only?
1. ![image](https://github.com/mdn/content/assets/373675/daa16acb-972e-4967-a91b-d5bd52f651a6)
  This is a "Content" request, so it fits, but it fits as much as the 1st choice.

**Comment by hamishwillee** [comment, marked as answer]
Please use the first one - "Content Issue".

@schalkneethling FYI. 3 and 4 above perhaps need a little clarification.
